## parameter for uav_command_pub
Controller_Test:
  Circle:
    Center_x: 0.0
    Center_y: 0.0
    Center_z: 1.0
    circle_radius: 1.0          ## m   半径越小越难追踪
    linear_vel: 1.0             ## m/s  速度越大越难追踪
    direction: 1.0              ##direction = 1 for CCW 逆时针, direction = -1 for CW 顺时针
  Eight:
    Center_x: 0.0
    Center_y: 0.0
    Center_z: 1.0
    radial: 2.0               ## m   半径越小越难追踪
    omega: 0.3                ## rad/s  角速度越大越难追踪
  Step:
    step_length: 2.0          ## m    长度越大越难追踪
    step_interval: 5.0




